.github/workflows/tests.yml

```diff
@@ -41,3 +85,21 @@ jobs:
           ( github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository )
         with:
           files: ./**/build/test-results/**/*.xml
+
+# Notify the channel about runs against develop or main that have failures, as PRs should have caught these first.
+  notify:
+    runs-on: ubuntu-latest
+    needs:
+      - unit-tests
+      - build-android-test-matrix-sdk
+      - build-android-test-app
+    if: ${{ (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/main' ) && failure() }}
+    steps:
+      - uses: michaelkaye/matrix-hookshot-action@v0.3.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          matrix_access_token: ${{ secrets.ELEMENT_ANDROID_NOTIFICATION_ACCESS_TOKEN }}
+          matrix_room_id: ${{ secrets.ELEMENT_ANDROID_INTERNAL_ROOM_ID }}
+          text_template: "Build is broken for ${{ github.ref }} 🎊: {{#each job_statuses }}{{#with this }}{{#if completed }}  {{name}} {{conclusion}} at {{completed_at}}, {{/if}}{{/with}}{{/each}}"
+          html_template: "Build is broken for ${{ github.ref }} 🎊: {{#each job_statuses }}{{#with this }}{{#if completed }}<br />{{icon conclusion }}{{name}} <font color='{{color conclusion }}'>{{conclusion}} at {{completed_at}} <a href=\"{{html_url}}\">[details]</a></font>{{/if}}{{/with}}{{/each}}"
```

Comment:
Emojis FTW! But here we have an error, so maybe use a "error" emoji?
Applicable to line 103 as well

Reply:
Ah, my bad, I hadn't read

> Add a notification step that triggers if the build fails on develop or main, and sends a confetti-filled ball to the channel to announce with pleasure that the build has failed and someone should do something about it.

Please ignore me!